tests.js

```diff
@@ -9,9 +9,47 @@
     return;
   }
 
-  function qpair(template, ...values) {
-    return [template, values];
+  // Set up a RegExp subclass so that we can test subclass creation.
+  function SubRegExp(source, flags) {
+    RegExp.call(this, source, flags);
```

Comment:
Not sure whether this makes sense at all while ES6 subclassing is not supported. Admittedly, `RegExp.call(this, source, flags);` throws the errors we would expect, but it doesn't modify `this` (neither in ES5 nor ES6).